lib/bundle/installer.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Bundle
+  module Installer
```

Comment:
Why the new class?

Reply:
I extracted it from [here](https://github.com/Homebrew/homebrew-bundle/pull/326/commits/fbe0b9eaec85f19a78f05a69234ee324ed9640bc#diff-d485b64f0c3589a2a3afb078040dde93L35). I must have forgotten to add it on that commit. Ugh. Got it here, though.

Moving it to a separate class seemed appropriate given the existence of other tool classes like this: Dumper and XDumper were a model, where X is one of the types. There was XInstaller but not an Installer. Conforming to that setup made the installer parts fall in line and, most importantly, removed installation from the responsibility of the DSL parser. I felt that the DSL parser should just turn the DSL into some model that can be acted on later: the entries.

Reply:
Ok, seems reasonable in this case. In future: can you split similar refactorings into a separate PR? It makes it harder to review when in an unrelated one. Thanks!

Reply:
Will do. It was the first thing I did wording toward this and then it ended up not being 100% necessary.